src/init.cpp

```diff
@@ -1398,7 +1401,7 @@ bool AppInitMain(const util::Ref& context, NodeContext& node, interfaces::BlockA
     ChainstateManager& chainman = *Assert(node.chainman);
 
     assert(!node.peerman);
-    node.peerman = std::make_unique<PeerManager>(chainparams, *node.connman, node.banman.get(),
+    node.peerman = std::make_unique<PeerManager>(chainparams, *node.connman, *node.addrman, node.banman.get(),
                                                  *node.scheduler, chainman, *node.mempool, ignores_incoming_txs);
```

Comment:
Because net_processing doesn't depend on node/context (and adding that would be a circular dependency between net_processing and node/context). In fact, a NodeContext object isn't even needed to construct a PeerManager object - see denialofservice_tests.cpp.

Reply:
ok